edx_proctoring/templates/proctoring/seq_proctored_exam_rejected.html

```diff
@@ -0,0 +1,55 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session was reviewed and did not pass requirements
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b class="failure"> Failed </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      Your session was reviewed by one of our proctors and some activities noted violate the
+      <a href="#">requirements and guidelines</a> needed for a successful review,
+    {% endblocktrans %}
+  </p>
+  <hr>
+  <p>
+    {% blocktrans %}
+      Please see <a href="{{progress_page_url}}">your progress in this course </a>
+      for your general course credit worthiness.
+    {% endblocktrans %}
+  </p>
+</div>
+<div class="footer-sequence border-b-0 padding-b-0">
+  <span> {% trans "Can I contest this review?" %} </span>
+  <p class="proctored-exam-option">
```

Comment:
Same comments/question as for line 31 in seq_proctored_exam_error.html
@chrisndodge @griffresch